src/garage/tf/algos/__init__.py

```diff
@@ -25,4 +28,7 @@
     'TNPG',
     'TRPO',
     'VPG',
+    'NPOTaskEmbedding',
```

Comment:
this should be TEPPO TETRPO, TENPO, etc.following the same pattern as RL2PPO, MAMLTRPO, etc.